packages/widgets/login/src/modules/sagas.js

```diff
@@ -81,12 +81,22 @@ export function* handleFailedResponse() {
 }
 
 export function* handleRedirect() {
-  const {redirectUrl} = yield select(inputSelector)
+  const redirectUrl = yield loadRedirectUrl()
   if (redirectUrl) {
     window.location.href = redirectUrl
   }
 }
 
+export function* loadRedirectUrl() {
+  const {_redirect_url: queryUrl} = queryString.fromQueryString(window.location.search)
+  if (queryUrl) {
+    return queryUrl
+  } else {
+    const {inputUrl} = yield select(inputSelector)
```

Comment:
question(blocking): is this still working after changing `redirectUrl` to `inputUrl`?